data/operators.csv

```diff
@@ -266,6 +266,7 @@ INTERSPACE-MK,cloud,,unsafe,200899,5200
 Monkeybrains,ISP,,unsafe,32329,5215
 BroadbandGibraltarLtd.,ISP,,unsafe,34803,5285
 AltusHost,cloud,,unsafe,51430,5306
+Kingston Communications,ISP,,unsafe,12390,5388
```

Comment:
AS12390 has signed all their prefixes. Suggest change to `partially safe` category.
```suggestion
Kingston Communications,ISP,signed,partially safe,12390,5388
```